src/router/router.ts

```diff
@@ -14,6 +14,7 @@ export function getVueRouter () {
     // set base URL for Vue Router
     base: sessionStorage.getItem('VUE_ROUTER_BASE'),
     routes,
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
'to' is defined but never used
'from' is defined but never used
'savedPosition' is defined but never used